ppa/archive/models.py

```diff
@@ -1206,126 +1206,69 @@ def index_item_type(cls):
         return "page"
 
     @classmethod
-    def page_index_data(cls, digwork):
+    def page_index_data(cls, digwork, gale_record=None):
         """Get page content for the specified digitized work from Hathi
-        pairtree and return data to be indexed in solr."""
+        pairtree and return data to be indexed in solr.
 
-        # TODO: how to share common fields/logic across sources?
+        Takes an optional Gale item record as returned by the Gale API,
+        to avoid loading API content twice. (Used on import)"""
```

Comment:
Method description should be updated to reflect that this method is used for both HathiTrust and Gale items (and in the future EEBO).